stdpopsim/catalog/BosTau/demographic_models.py

```diff
@@ -1,31 +1,34 @@
 import msprime
-
 import stdpopsim
 
 _species = stdpopsim.get_species("BosTau")
 
 # Inferring Demography from Runs of Homozygosity in Whole-Genome Sequence,
 # with Correction for Sequence Errors.
 _MacLeodEtAl = stdpopsim.Citation(
-    doi="https://doi.org/10.1093/molbev/mst125",
-    year="2013",
     author="MacLeod et al.",
+    year=2013,
+    doi="https://doi.org/10.1093/molbev/mst125",
+    reasons={
+        stdpopsim.CiteReason.GEN_TIME,
+        stdpopsim.CiteReason.POP_SIZE,
+        stdpopsim.CiteReason.DEM_MODEL,
+    },
 )
 
 
-def _HolsteinFriesan_1M13():
-    id = "HolsteinFriesian_1M13"
+def _HolsteinFriesian_IM13():
+    id = "HolsteinFriesian_IM13"
```

Comment:
As Peter suggested, this id needs to have `_1M13` (one-em-thirteen) rather than `_IM13` (eye-em-thirteen). The `1` refers to the number of populations in the model.

Reply:
Done!